src/features/companyCertification/companyCertificateApiSlice.tsx

```diff
@@ -25,6 +25,11 @@ export type PaginationData = {
   page: number
 }
 
+export enum CompanyCertificateStatusEnum {
+  ACTIVE = 'ACTIVE',
```

Comment:
This enum is never used in this PR. If It is not needed here, please remove this.